.idea/libraries/testng_6_7.xml

```diff
@@ -0,0 +1,9 @@
+<component name="libraryTable">
```

Comment:
.idea folder must be excluded from the commit.

Reply:
added file to .gitignore